src/utils/getSortedPeople.ts

```diff
@@ -0,0 +1,48 @@
+import { SorterTypes, SorterOrder } from './enums/sortedEnums';
+import { Person } from '../types';
+
+const sortPeopleByName =
+  (order: null | SorterOrder) => (a: Person, b: Person) =>
+    order === SorterOrder.DESC
+      ? b.name.toLowerCase().localeCompare(a.name.toLowerCase())
+      : a.name.toLowerCase().localeCompare(b.name.toLowerCase());
```

Comment:
DRY, It looks like we can move this fragment into a separate helper function`b.name.toLowerCase().localeCompare(a.name.toLowerCase())`